packages/core/src/components/Search/Search.tsx

```diff
@@ -116,8 +118,10 @@ const Search = forwardRef(
         inputRole={searchResultsContainerId ? "combobox" : undefined}
         aria-label={inputAriaLabel}
         aria-busy={loading}
-        aria-owns={searchResultsContainerId}
+        aria-controls={ariaExpanded ? searchResultsContainerId : undefined}
         aria-activedescendant={currentAriaResultId}
+        aria-haspopup={ariaHasPopup}
+        aria-expanded={ariaExpanded}
```

Comment:
this should be set for elements that have child elements that are either shown or hidden, same in this case - Search does not have child elements